src/session.py

```diff
@@ -66,12 +66,12 @@ def print_content(self, key: str, is_secure: bool) -> None:
         PasswordDataIO.print(self.content[key], is_secure=is_secure)
 
     def destroy(self, name: str) -> None:
-        pass
-        #pathfile = self.passpath+'/'+self.name
-        #if os.path.exists(pathfile):
-        #   os.remove(pathfile)
-        #else:
-        #   Io.print('File does not exist')
+        pathfile = self.passpath + '/' + name
+        if os.path.exists(pathfile):
+            os.remove(pathfile)
+            self.files = os.listdir(CRYPTPATH)
```

Comment:
Could replace by `self.files.remove(pathfile)`
This should be equivalent and not require an os call to list the remaining files.